Tools/msi/README.txt

```diff
@@ -67,18 +67,18 @@ similarly named file in PCbuild.)
 
 One of the dependencies used in builds is WiX, a toolset that lets developers
 create installers for Windows Installer, the Windows installation engine. WiX
-has a dependency on the Microsoft .NET Framework Version 3.5 (which may not be
-configured on recent versions of Windows, such as Windows 10). If you are
-building on a recent Windows version, use the Control Panel (Programs | Programs
-and Features | Turn Windows Features on or off) and ensure that the entry
-".NET Framework 3.5 (includes .NET 2.0 and 3.0)" is enabled.
-
-For Python 3.11.x and above, enable "Microsoft .NET Framework 4.8 Advanced Services"
-instead of "Microsoft .NET Framework Version 3.5" available for Windows 10 and above.
-Also make sure "MSVC v143 - VS 2022 C++ ARM64 build tools" are selected under
-"Desktop Development with C++" in "Visual Studio installer" even if you are not
-building on ARM64 along with other x64 related v143 build tools. This is because for
-3.11.x and above we have upgraded to Wix-3.14.
+has a dependency on the Microsoft .NET Framework 4.8 Advanced Services (which
+may not be configured on recent versions of Windows, such as Windows 10 or
+later). If you are building on these recent versions, use the Control Panel
+(Programs | Programs and Features | Turn Windows Features on or off) and
+ensure that the entry "Microsoft .NET Framework 4.8 Advanced Services" is
+enabled.
```

Comment:
We shouldn't need any Advanced Services - .NET Framework 4.8 should be always available. If WiX has dropped their need for .NET 3.5 then this paragraph can probably go away.